axelor-supplychain/src/main/resources/i18n/messages_fr.csv

```diff
@@ -282,6 +283,7 @@
 "No stock location valid. Please uncheck the chosen stock location 'is not in MRP'.","Pas d'emplacement de stock valide. Veuillez décocher le champ 'Ne pas prendre en compte dans le CBN' sur l'emplacement de stock sélectionné.",,
 "Not enough quantity are available for reservation for product %s (%s)",,,
 "Note",,,
+"Number of Days",,,
```

Comment:
No translation provided for  'Number of Days'
Change it to 'Number of days' so existing translation will get used

Reply:
Changed the title to 'Number of days'.